src/waveform/visualplayposition.h

```diff
@@ -14,6 +14,12 @@ typedef void VSyncThread;
 class VSyncThread;
 #endif
 
+enum class SlipModeStates {
```

Comment:
```suggestion
enum class SlipModeState {
```
I think singular is used in other places